WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -392,7 +392,7 @@ private void renderPost(final int position, final ReaderPostViewHolder holder) {
         };
 
         Function2<Long, Long, Unit> onVideoOverlayClicked = (postId, blogId) -> {
```

Comment:
We're not using (postId, blogId) - can we remove these?

Reply:
We are not using them in the adapter because the renderPost method has access to a ReaderPost reference. However, I believe the VM won't have access to such reference so we'll need them (eg [here](https://github.com/wordpress-mobile/WordPress-Android/blob/9e20bd906574062a829071b0a15f5843c9292b36/WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt#L130)). I'll clean-up all these listeners after the actions are actually implemented.